include/dlaf/common/pipeline.h

```diff
@@ -42,10 +43,25 @@ class Pipeline {
   ///
   /// @return a sender that will become ready as soon as the previous user releases the resource.
```

Comment:
Document the precondition (in other places as well)

Reply:
https://github.com/eth-cscs/DLA-Future/pull/881/commits/c47688b2ff2394015fdea3c08259dc332e83e805